app/models/user.py

```diff
@@ -57,10 +57,11 @@ class User(UserMixin, db.Model):
     def __init__(self, **kwargs):
         super(User, self).__init__(**kwargs)
         if self.role is None:
```

Comment:
It is. I was just looking through this logic trying to figure out how the invite form works. In admin/views.py:

    def invite_user():
        """Invites a new user to create an account and set their own password."""
        form = InviteUserForm()
        if form.validate_on_submit():
            user = User(role=form.role.data,
                        first_name=form.first_name.data,
                        last_name=form.last_name.data,
                        email=form.email.data)
            db.session.add(user)
            db.session.commit()
            ...

See how it instantiates User?